refarch-integrations/refarch-s3-integration/refarch-s3-integration-client-core/src/main/java/de/muenchen/refarch/integration/s3/client/exception/FileSizeValidationException.java

```diff
@@ -1,6 +1,11 @@
 package de.muenchen.refarch.integration.s3.client.exception;
 
+import java.io.Serial;
+
 public class FileSizeValidationException extends RuntimeException {
+    @Serial
+    private static final long serialVersionUID = 1L;
```

Comment:
Won't comment any further for now. But we can still talk about if its needed for exceptions or not. But we should implement equally in refarch-templates and refarch.

Reply:
Done